doc/usermanual/lighttable/panels/export.xml

```diff
@@ -790,6 +807,79 @@
+        into the exported image.
+      </para>
+
+      <para>
+        Metadata configuration is saved along with other export selected parameters to your favorites presets.
+      </para>
+
+      <para>
+        <graphic fileref="lighttable/images/panel_exportselected-metadata-config.png" scalefit="1" width="80%" align="center" />
+        The left part lets you select the groups of metadata to be exported.
+        The  entry <emphasis>metadata</emphasis> corresponds to metadata defined in metadata editor.
+        Under entry <emphasis>tags</emphasis> you can request the exportation of private tags,
+        synonyms and omit to include the path elements of hierarchical tags. Remember that a tag set up as a category is never exported.
+      </para>
+
+      <para>
+        The right part lets you define a formula per metadata. What you define here
+        has priority on the left settings.
+        The first column identifies the metadata. The second one is editable and defines the metadata calculation.
+        Letting the formula empty prevents to export this specific metadata (Exif.GPSInfo.GPSVersionID in the above example).
+        Otherwise you can use any of the variables and rules defined above for filename.
+        Click on <emphasis>enter</emphasis> to validate the formula.
+      </para>
+
+      <para>
+        Use the icon <quote>-</quote> to remove a metadata from the list, the
+        icon <quote>+</quote> to add a new one from the predefined list:
+        <graphic fileref="lighttable/images/panel_exportselected-metadata-selection.png" scalefit="1" width="40%" align="center" />
+        Double-click or click on button <emphasis>add</emphasis> to add a metadata to the configuration list.
+      </para>
+
+      <para>
+        The formulas allow you to get the most of the tagging function. You can virtually define
+        all the metadata you need to qualify your images in tagging and, thanks to these
+        formulas, export the values in the xmp or iptc tags of your choice. The exported tags
+        can be different from one export to the other depending on the destination of the images.
+        Just remember that tags set as <emphasis>category</emphasis> are not exported.
+        Tags and Categories are displayed separately in image informations.
+      </para>
+
+      <para>
+        Example 1. A first level tag called <emphasis>places</emphasis> set as category followed
+        by four levels, country, region, city and location: <code>places|France|Nord|Lille|rue Nationale</code>.
+        The formulas <code>$(CATEGORY0(places))</code>, <code>$(CATEGORY1(places))</code>,
+        <code>$(CATEGORY2(places))</code> and <code>$(CATEGORY3(places))</code>
+        retrieve the four values (when exist) which can be assigned separately to the xmp tags you need.
+        Image informations display the four values in tags list. They can also be exported as simple tags <code>$(TAGS)</code>, unlike
```

Comment:
Okay, took me a little while to hunt down exactly how this thing was working (I don't tag my pictures, had never used the feature :) ), so please bear with me if my rephrasing is not correct... but I believe these sentences are a bit confusing and/or too terse here, so I'm suggesting an alternative.
"`The four values are displayed in image information as a tag list. They can also be exported as simple tags with <code>$(TAGS)</code>. Since <emphasis>places</emphasis> is a category, it is not displayed by these means.`"

Reply:
I agree the text is not clear. I've tried to be more explicit.